getting-started/scaling-out.md

```diff
@@ -0,0 +1,128 @@
+## Working with Data Nodes
+
+Data nodes act as the containers of the hypertable partitions and are
+necessary to be able to create distributed hypertables. Data nodes are
+added to the current database using [`add_data_node`][add_data_node]
+and removed using [`delete_data_node`][delete_data_node].
+
+Note that data nodes are added as objects locally and you should
+already have a running database server on some host. When creating the
+data node, you should provide a (local) name to use when referring to
+the data node as well as the host where the hypertable partition
+should be stored.
+
+```sql
+SELECT add_data_node('node1', host => 'dn1.example.com');
+SELECT add_data_node('node2', host => 'dn2.example.com');
+```
+
+Deleting a data node is done in a similar manner.
+
+```sql
+SELECT delete_data_node('node1');
+```
+
+Note that a data node cannot be deleted if it contains data for a
+hypertable. You have to ensure either that there is no data on the
+data node, or that the data is replicated to other nodes.
+
+### Information Schema for Data Nodes
+
+The data nodes that have been added to the database can be found by
+querying the `timesscaledb_information.data_node`.
+
+| Column            | Type    | Description                                               |
+|-------------------|---------|-----------------------------------------------------------|
+| `node_name`       | name    | Data node name                                            |
+| `owner`           | oid     |                                                           |
+| `options`         | text[]  | Options used when creating the data node                  |
+| `node_up`         | boolean | Data node responds to ping                                |
+| `num_dist_tables` | bigint  | Number of distributed hypertables that use this data node |
+| `num_dist_chunks` | numeric | Total number of chunks stored in data node                |
+| `total_dist_size` | text    | Total amount of data stored in data node                  |
+
+## Working with Distributed Hypertables
+
+As previously mentioned, distributed hypertables are similar to normal
+hypertable with the difference that distributed hypertables will
+partition the data and store it on remote data nodes. This means that
+to create a hypertable, you need to have added at least one data node
+before starting to create distributed hypertables.
+
+### Creating Distributed Hypertables
+
+As with normal hypertables, you have to start by creating a regular
+table to hold the data, then transform it into a distributed
+hypertable. In contrast to when creating a normal hypertable, we use
+the function
+[`create_distributed_hypertable`][create_distributed_hypertable].
+
+```sql
+CREATE TABLE conditions (
+  time        TIMESTAMPTZ       NOT NULL,
+  location    TEXT              NOT NULL,
+  temperature DOUBLE PRECISION  NULL,
+  humidity    DOUBLE PRECISION  NULL
+);
+
+SELECT create_distributed_hypertable('conditions', 'time');
```

Comment:
We should include a space partitioning dimension in this example as this is the recommended configuration.